common/util/vector_tree.h

```diff
@@ -374,9 +523,9 @@ class VectorTree : private _VectorTreeImpl {
 
   const this_type* Parent() const { return parent_; }
 
-  subnodes_type& Children() { return children_; }
+  auto& Children() { return children_; }
```

Comment:
Since we know the return type, why `auto` ?

Reply:
Replaced with `VectorTreeChildrenList`.